web-server/v0.4/src/utils/parse.js

```diff
@@ -207,6 +209,7 @@ export const parseClusteredIterations = (clusteredIterations, clusterLabels, sel
   const clusteredGraphData = [];
   const graphKeys = [];
   const tableData = [];
+  let noOfIteration = 0;
```

Comment:
Can we consider naming this variable `maxClusterLength` to better describe this value store as the max number of iterations found out of all clusters?